install.sh

```diff
@@ -259,27 +271,25 @@ install_packages( )
   if [ -z "$CMAKE_VERSION" ] || $(dpkg --compare-versions $CMAKE_VERSION lt 3.16.8); then
     if $update_cmake == true; then
       library_dependencies_ubuntu+=("wget")
-      library_dependencies_centos_rhel+=("wget")
-      library_dependencies_centos_rhel_8+=("wget")
+      library_dependencies_centos+=("wget")
+      library_dependencies_centos8+=("wget")
       library_dependencies_fedora+=("wget")
       library_dependencies_sles+=("wget")
     fi
   fi
 
-  local client_dependencies_ubuntu=( "gfortran" )
-  local client_dependencies_centos=( "devtoolset-7-gcc-gfortran" )
-  local client_dependencies_centos8=( "gcc-gfortran" )
-  local client_dependencies_fedora=( "gcc-gfortran" )
-  local client_dependencies_sles=( "pkg-config" "dpkg" )
+  if [[ "${build_clients}" == true ]]; then
+    library_dependencies_ubuntu+=( "gfortran" )
+    library_dependencies_centos+=( "devtoolset-7-gcc-gfortran" )
+    library_dependencies_centos8+=( "gcc-gfortran" )
```

Comment:
So rhel and centos are using identical deps any reason you didn't want to call these centos_rhel as before?

Reply:
Well they may diverge with stream if not 8.6 so doesn't hurt to be clearer it is used for both.